tests/timezones/tests.py

```diff
@@ -607,7 +607,8 @@ class SerializationTests(SimpleTestCase):
     # - JSON supports only milliseconds, microseconds will be truncated.
     # - PyYAML dumps the UTC offset correctly for timezone-aware datetimes,
     #   but when it loads this representation, it subtracts the offset and
-    #   returns a naive datetime object in UTC (https://pyyaml.org/ticket/202).
+    #   returns a naive datetime object in UTC
+    #   (https://github.com/yaml/pyyaml/pull/113).
```

Comment:
I would remove this reference. IMO adding a note to the ticket [18867](https://code.djangoproject.com/ticket/18867) about open PR in PyYAML would be more appropriate.

Reply:
Added a comment to django ticket and referred the ticket number in code comment.